src/components/calculatorCards/bmrCalculator.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useState } from "react";
+import { Button, Card, Col, Form, Row, DropdownButton, Dropdown } from "react-bootstrap";
+import "./calculatorStyles.css"
+
+let age: number = 0;
+let height: number = 0;
+let weight: number = 0;
+
+ // BMR (kcal / day) = 10 * weight (kg) + 6.25 * height (cm) – 5 * age (y) + s (kcal / day)
+ // where s is +5 for males and -161 for female
+ // formula taken from https://www.omnicalculator.com/health/bmr#what-is-bmr-bmr-definition
+ // amr = active metabolic rate
+ // Sedentary (little or no exercise): AMR = BMR x 1.2
+ // Lightly active (exercise 1–3 days/week): AMR = BMR x 1.375
+ // Moderately active (exercise 3–5 days/week): AMR = BMR x 1.55
+ // Active (exercise 6–7 days/week): AMR = BMR x 1.725
+ // Very active (hard exercise 6–7 days/week): AMR = BMR x 1.9
+ // taken from https://www.verywellfit.com/how-many-calories-do-i-need-each-day-2506873
+
+enum activityLevel {
+    None = 1,
+    Sedentary = 1.2,
+    Lightly = 1.375,
+    Moderately = 1.55,
+    Active = 1.725,
+    VeryActive = 1.9
+}
+
+function bmrCalculate(gender: boolean, activity: activityLevel) {
+  let s: number = 0;
```

Comment:
This function could be just a return gender
    ? Math.round((10 * weight + 6.25 * height - 5 * age + 5) * activity)
    : Math.round((10 * weight + 6.25 * height - 5 * age - 168) * activity);

Reply:
ty will do